cli.js

```diff
@@ -101,39 +99,50 @@ async function main() {
       name,
     })
   }
+  return lttfPlugins;
+}
 
-  switch (argv._[0]) {
-    case 'output':
-      await output(lttfPlugins);
-
-      break;
-
-    case 'list':
-      if (!argv.o && !argv.stdout) {
-        console.error('You must provide an output path to `list` with -o or pass --stdout');
-      }
-
-      // eslint-disable-next-line
-      const listResult = await list(lttfPlugins);
+program.name('lint-to-the-future')
+  .description('A modern way to progressively update your code to the best practices')
+  .version('1.2.0');
+  program.command('output')
+    .description('Generates a dashboard webpage to help track your disables')
+    .requiredOption('-o, --output <path>', 'Output path for dashboard webpage')
+    .option('--rootUrl', 'Required if the dashboard is not hosted on your servers rootUr')
+    .option('--previous-results <path|url>', 'This should be a path or URL to the previous data.json file that was generated when this command was last run')
+    .action(async ({output: outputPath, rootUrl, previousResults}) => {
+      let lttfPlugins = await getLttfPlugins();
+      output(lttfPlugins, outputPath, rootUrl, previousResults);
+    })
 
-      if (argv.stdout) {
-        console.log(listResult);
-      }
+program.command('list')
+  .description('Chris should write a good description here')
```

Comment:
I couldnt think of a great description so let me know if you have any ideas.

Reply:
lol I should have left it in there 😂 

but alas I have updated it 👍  thanks for letting me know 🎉